### ISSUE TITLE
Accessibility Review nieuwe website

### ISSUE THREAD

**Issue by petergoes** [nl-design-system/documentatie#4667]
## Relevante viewports:
- [ ] minimum: `320px`
- [ ]  breakpoints:
  - [ ]  width >= `768px`: `--medium-and-up`
  - [ ]  width >= `1140px`: `--large-and-up`
  - [ ]  width >= `1280px`: `--x-large-and-up`

## Relevante varianten:
- [ ] forced colors mode
- [ ] dark mode
- [ ] light mode
- [ ] wel én geen javascript
  - even noteren wat niet werkt zonder javascript
- [ ] wel en geen CSS
 
- overzicht van alle design system en content (markdown) componenten die gebruikt zijn, en een voorbeeld van welke pagina ze te vinden zijn.

## Overzicht van paginatypes.

- Homepage
	- [homepage](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/)
- Content pagina
	- [/handboek/componenten-vinden/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/handboek/componenten-vinden/)
	- [/richtlijnen/content/tekstopmaak/kleurgebruik-in-tekst/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/richtlijnen/content/tekstopmaak/kleurgebruik-in-tekst/)
	- [/community/community-sprints/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/community/community-sprints/)
	- [/handboek/estafettemodel/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/handboek/estafettemodel/)
	- [/woordenlijst/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/woordenlijst/)
- Overview pagina
	- [/componenten/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/componenten/)
	- [/project/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/project/)
	- [/richtlijnen/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/richtlijnen/)
	- [/voorbeelden/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/voorbeelden/)
	- [/handboek/designer/introductie/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/handboek/designer/introductie/)
- Overview pagina (gegenereerd)
	- [/community/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/community/)
	- [/handboek/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/handboek/)
	- [/community/events/overzicht/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/community/events/overzicht/)
	- [/handboek/huisstijl-vastleggen/overzicht/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/handboek/huisstijl-vastleggen/overzicht/)
	- [/handboek/organisatie/overzicht/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/handboek/organisatie/overzicht/)
- Component detail pages
	- [/accordion/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/accordion/)
	- [/button/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/button/)
	- [/dialog/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/dialog/)
	- [/tabs/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/tabs/)
	- [/checkbox/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/checkbox/)
- WCAG pagina
	- [/wcag/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/wcag/) 
	- [/wcag/1.1.1/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/wcag/1.1.1/)
	- [/wcag/1.3.1/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/wcag/1.3.1/)
	- [/wcag/1.4.1/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/wcag/1.4.1/)
	- [/wcag/2.1.1/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/wcag/2.1.1/)
	- [/wcag/2.4.1/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/wcag/2.4.1/)
- Overig
	- [/CHANGELOG/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/CHANGELOG/)
	- [/sitemap/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/sitemap/)
	- [/toegankelijkheidsverklaring](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/toegankelijkheidsverklaring)
	- [/slack](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/slack)
	- [/privacyverklaring](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/privacyverklaring)

## Overzicht van media.
  - geen audio only atm.
  - wel video's in de website van screen reader recordings van Rian.

## 3rd party plugins die je gebruikt.
  - bijv: youtube.
  - laposta.

## Wat is in scope.
- nldesignsystem.nl is in scope.
- maar ook content (titel, caption inhoud van de youtube video) niet de hele youtube website.
- talen:.
	- alleen nederlands.
	- engelstalige pagina's hier.
		- [/events/design-systems-week/en/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/events/design-systems-week/en/)
		- [/events/design-systems-week-2026/en/program/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/events/design-systems-week-2026/en/program/)
		- [/events/design-systems-week-2026/en/timetable/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/events/design-systems-week-2026/en/timetable/)
		- [/events/design-systems-week-2024/en/program/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/events/design-systems-week-2024/en/program/)
		- [/events/design-systems-week-2025/en/program/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/events/design-systems-week-2025/en/program/)
	- tbd: hebben we mixed content pagina's? ja: <span lang="en"> linkjes sowieso veel, misschien ook wat blokken tekst in Engels.

## Doel van de audit:.
  - WCAG 2.2 compliance.
  - maximaal toegankelijk, we horen graag over best practice tips.
  - [https://nldesignsystem.nl/wcag/2.5.5/](https://nldesignsystem.nl/wcag/2.5.5/)
  - [https://nldesignsystem.nl/wcag/2.4.13/](https://nldesignsystem.nl/wcag/2.4.13/)
  - [https://nldesignsystem.nl/wcag/2.4.12/](https://nldesignsystem.nl/wcag/2.4.12/)
  - [https://nldesignsystem.nl/wcag/2.4.8/](https://nldesignsystem.nl/wcag/2.4.8/)
  - [https://nldesignsystem.nl/wcag/2.3.2/](https://nldesignsystem.nl/wcag/2.3.2/)
  - [https://nldesignsystem.nl/wcag/2.1.3/](https://nldesignsystem.nl/wcag/2.1.3/)
  - [https://nldesignsystem.nl/wcag/1.4.9/](https://nldesignsystem.nl/wcag/1.4.9/)
  - [https://nldesignsystem.nl/wcag/1.4.8/](https://nldesignsystem.nl/wcag/1.4.8/)

pagina's om mee te geven:
- misschien hidden testpagina's zoals die voor code
- acid test pagina voor content, met bijv elk mogelijk contentblok

## Essential user stories:
- sign up formulier voor diverse dingen, 1 flow testen is voldoende
	- [/community/sluit-je-aan/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/community/sluit-je-aan/)
- documentatie lezen in handboek/richtlijnen/voorbeelden/project
	- [/handboek/huisstijl/basis-tokens](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/handboek/huisstijl/basis-tokens)
	- [/handboek/designer/figma-structuur](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/handboek/designer/figma-structuur)
- video bekijken in community / heartbeat
	- [/events/heartbeat/videos](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/events/heartbeat/videos)
- contact opnemen per e-mail
	- [/project/kernteam/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/project/kernteam/)
- toegankelijkheidsverklaring vinden (is die ook in de sitemap?)
- privacyverklaring vinden (is die ook in de sitemap?)
- documentatie vinden via zoeken
	- [/zoeken/](https://documentatie-next-git-release-website-next-nl-design-system.vercel.app/zoeken/)

## Welke van de volgende Web Platform Techniques worden in de website gebruikt?

- CSS ✅
- HTML ✅
- SVG ✅
- WAI-ARIA ✅
- Web Components ✅
- Video ✅
- Audio ❌
- Canvas ❌
- MathML ❌
- WebGL ❌
- contentEditable ❌

## frameworks:
- NL Design System
- Astro
- React
- Markdown

**Comment by Robbert** [comment]
Misschien wat dubbele dingen,
maar ik moest nog denken aan:

[edit van @petergoes]: Ik heb de onderstaande lijst doorgelopen en daar waar nodig verwerkt in de issue beschrijving

- timetable pagina https://nldesignsystem.nl/events/design-systems-week-2026/tijdschema/
- pagina met navigatie cards https://nldesignsystem.nl/community/
- component overzicht https://nldesignsystem.nl/componenten/
- component detail https://nldesignsystem.nl/button/
- video overzicht heartbeat https://nldesignsystem.nl/events/heartbeat/videos/
- programma design systems week https://nldesignsystem.nl/events/design-systems-week-2025/programma/
- video overzicht design systems week https://nldesignsystem.nl/events/design-systems-week-2024/programma/
- contact info cards voor kernteam https://nldesignsystem.nl/wcag/
- zoekresultaten
- aanmeldformulier met alleen form https://nldesignsystem.nl/community/belangenorganisaties/aanmelden/
- aanmeldformulier in grotere pagina https://nldesignsystem.nl/community/sluit-je-aan/
- aanmelden bedankt pagina https://nldesignsystem.nl/community/sluit-je-aan/bedankt/
- wcag pagina (deze is met video en code) https://nldesignsystem.nl/wcag/2.4.1/
- richtlijn pagina met do / don't / code voorbeeld https://nldesignsystem.nl/richtlijnen/formulieren/visueel-ontwerp/placeholder-goed-zichtbaar/
- wcag overview page https://nldesignsystem.nl/wcag/
- sitemap https://nldesignsystem.nl/sitemap
- homepage https://nldesignsystem.nl/
- estafettemodel uitleg widget https://nldesignsystem.nl/handboek/estafettemodel/
- toegankelijkheidsrapport https://nldesignsystem.nl/toegankelijkheidsverklaring
- pagina met infographics https://nldesignsystem.nl/handboek/designer/figma-structuur/
- technische documentatie https://nldesignsystem.nl/handboek/huisstijl/basis-tokens/
- overzicht van tokens https://nldesignsystem.nl/handboek/designer/figma-structuur/
- pagina met mix tussen interne en externe links https://nldesignsystem.nl/slack/
- privacyverklaring als essentiele user story https://nldesignsystem.nl/privacyverklaring/
